app-libs/stf/src/trusted_call.rs

```diff
@@ -384,6 +378,36 @@ impl ExecuteCall for TrustedCallSigned {
 			},
 
 			 */
+			TrustedCall::ceremonies_attest_attendees(
+				who,
+				cid,
+				number_of_participants_vote,
+				attestations,
+			) => {
+				let origin = ita_sgx_runtime::Origin::signed(who);
+
+				if pallet_encointer_scheduler::Pallet::<ita_sgx_runtime::Runtime>::current_phase()
+					!= CeremonyPhaseType::Attesting
+				{
+					return Err(Self::Error::Dispatch(
+						"attendees attestation can only be done during attesting phase".to_string(),
+					))
+				}
```

Comment:
Doesn't the pallet already throw an error for this?

Reply:
Yes it does. Actually, I also did something similar in other truted calls:
https://github.com/encointer/community-sidechain/blob/53c7400caba1292775e56668dc865be230957177/app-libs/stf/src/trusted_call.rs#L475
and https://github.com/encointer/community-sidechain/blob/53c7400caba1292775e56668dc865be230957177/app-libs/stf/src/trusted_call.rs#L396
Shall I remove them?

Reply:
OK